packages/liquidator/index.js

```diff
@@ -174,6 +184,15 @@ async function run({
       liquidatorConfig
     });
 
+    logger.debug({
+      at: "Liquidator#index",
+      message: "Liquidator initialized",
+      collateralDecimals: Number(collateralTokenDecimals),
+      syntheticDecimals: Number(syntheticTokenDecimals),
+      priceFeedDecimals: Number(priceFeed.getPriceFeedDecimals()),
```

Comment:
Is this the only case where a pricefeed's `getPriceFeedDecimals()` is used? If so, how do you anticipate this method can be used.